front_end/android/app/src/main/java/com/front_end/MainApplication.java

```diff
@@ -4,6 +4,9 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.oblador.vectoricons.VectorIconsPackage;
+import com.oblador.vectoricons.VectorIconsPackage;
+import com.oblador.vectoricons.VectorIconsPackage;
 import com.facebook.react.ReactInstanceManager;
```

Comment:
Why are there 3 of the same import calls?

Reply:
seconded